cmd/adhoc.go

```diff
@@ -75,6 +75,10 @@ func (a *Adhoc) NewServer(userConfig legacy.UserConfig, config legacy.Config) (S
 		GithubAppKeyFile: userConfig.GithubAppKeyFile,
 		GithubAppSlug:    userConfig.GithubAppSlug,
 		GlobalCfg:        globalCfg,
+		ValidationConfig: neptune.ValidationConfig{
```

Comment:
nit: rather than inject default, can we just modify the `NewTerraform` starter logic to check for nil values and skip setting it up?